Source/Scene/Cesium3DTile.js

```diff
@@ -77,6 +77,7 @@ define([
      * @constructor
      */
     function Cesium3DTile(tileset, baseUrl, header, parent) {
+        this._tileset = tileset;
```

Comment:
Instead of keeping this circular reference, can we just pass `tileset.modelMatrix` to `updateTransform`?  Or break it into two functions `updateTransform` and `updateRootTransform` that call a common private function?